src/calibrate_all.jl

```diff
@@ -54,7 +55,13 @@ function calibrate_all(data::LegendData, sel::AnyValiditySelection, datastore::A
     
     maximum_with_init(A) = maximum(A, init=zero(eltype((A))))
 
-    is_valid_hit(trig_chs::AbstractVector{<:Int}, hit_channels::AbstractVector{<:Int}) = all(x -> x in hit_channels, trig_chs)
+    is_valid_trig(trig_chs::AbstractVector{<:Int}, hit_channels::AbstractVector{<:Int}) = all(x -> x in hit_channels, trig_chs)
+    
+    hit_properties = get_ged_evt_is_valid_hit_properties(data, sel)
+    is_valid_hit = trues(length(ged_events_pre.channel))
+    for prop in hit_properties
+        is_valid_hit .&= all.(map.(isfinite, (getindex.(getproperty(ged_events_pre, prop), trig_e_ch))))
+    end
```

Comment:
Just out of my mind: does this work?
```suggestion
    is_valid_hit = all.([all.(map.(isfinite, (getindex.(getproperty(ged_events_pre, prop), trig_e_ch)))) for prop in get_ged_evt_is_valid_hit_properties(data, sel)])
```